docs/examples/diagnostic_positions_exception.cpp

```diff
@@ -0,0 +1,29 @@
+#include <iostream>
+
+#define JSON_DIAGNOSTIC_POSITIONS 1
+#include <nlohmann/json.hpp>
+
+using json = nlohmann::json;
+
+/* Demonstration of type error exception with diagnostic postions support enabled */
+int main()
+{
+    //Invalid json string - housenumber type must be int instead of string
+    std::string json_invalid_string = R"(
+    {
+        "address": {
+            "street": "Fake Street",
+            "housenumber": "1"
+        }
+    }
+    )";
+    json j = json::parse(json_invalid_string);
+    try
+    {
+        int housenumber = j["address"]["housenumber"];
```

Comment:
@nlohmann Does it make sense to run Codacy on the examples?  If so, should this just remove the assignment?

Reply:
I have no idea if I can deactivate Codacy for specific folders. I'd rather see this issue fixed - maybe by `std::cout << j["address"]["housenumber"]` or something.

Reply:
Since the intent is to just show the exception `j["address"]["housenumber"];` may be enough, but then you might get a warning about a discarded return value, so the `std::cout` is probably better.